workflows/sarek.nf

```diff
@@ -252,6 +264,7 @@ workflow SAREK {
     intervals_bed_combined_gz     = intervals_bed_combined_gz_tbi.map{ bed, tbi -> [bed]}.collect() // one file containing all intervals interval.bed.gz file
     intervals_for_preprocessing   = (!params.wes || params.no_intervals) ? [] : PREPARE_INTERVALS.out.intervals_bed //TODO: intervals also with WGS data? Probably need a parameter if WGS for deepvariant tool, that would allow to check here too
 
+    // TODO: needs to figure something out when intervals are made out of the fasta_fai file
```

Comment:
this still relevant with the intervals sw?

Reply:
yes, because I can't get this number out of a workflow, or else it's channel dependent, not a variable and future declared

Reply:
ugh right, I forgot that 🙄